src/main/java/org/fungover/storm/httphandler/Response.java

```diff
@@ -0,0 +1,50 @@
+package org.fungover.storm.httphandler;
+
+import java.util.Map;
+
+public class Response {
+    private Map<String, String> headers;
+    private int statusCode;
+    private byte[] body;
+
+    public Response() {
+        this.reset();
+    }
+
+    public void reset() {
+        this.statusCode = 0;
+        this.body = null;
```

Comment:
If headers is initialized in the constructor then we can also reset body & headers in the same way as in Response.java and avoid assigning _null_.

```
this.body = new byte[0];
this.headers.clear();
```

Reply:
Good suggestion! I think I made this change beforehand but I must thave forgotten to commit :)